src/utils/illustration.js

```diff
@@ -521,17 +539,20 @@ const data = [{
 	strings: [
 		'Lecture',
 		'Seminar',
+		'Conference',
 		// TRANSLATORS This string is used for matching the event title to an illustration
 		t('calendar', 'Lecture'),
 		// TRANSLATORS This string is used for matching the event title to an illustration
 		t('calendar', 'Seminar'),
+		// TRANSLATORS This string is used for matching the event title to an illustration
+		t('calendar', 'Conference'),
 	],
 	illustrationNames: [
 		'professor',
 	],
 }, {
 	strings: [
-		'Photograph',
+		'Photograph', 'Photo',
```

Comment:
Put 'Photo' on a new line please

Reply:
As this one is just a shortened version, I thought it would make sense to specifically not put it on a new line.

Note that there also isn't a translation for it either, for the same reason. Should there be one?

Reply:
Ah, that makes sense in that case. Nevermind then. :)